src/components/MessageList/MessageList.tsx

```diff
@@ -142,25 +298,29 @@ const MessageList = (props) => {
         <DefaultMessage
           actionSheetStyles={actionSheetStyles}
           AttachmentFileIcon={AttachmentFileIcon}
-          groupStyles={message.groupStyles}
+          groupStyles={message.groupStyles as GroupType[]}
```

Comment:
i don't think we'd need this casting if the message was typed as above, so not sure which way we need to go

Reply:
it's this way to get around the Immutable crap